blocklyc.html

```diff
@@ -20,35 +20,29 @@
   DEALINGS IN THE SOFTWARE.
 -->
 
-<html>
+
+
+
+
+
+
+<!DOCTYPE html>
+<html lang="en">
     <head>
         <meta charset="utf-8">
         <meta name="base" content="">
         <meta name="cdn" content="src/">
-        <meta name="user-auth" content="true">
+        <meta name="projectlink" content="">
         <meta name="isOffline" content="true">
-        <meta name="in-demo" content="demo">
-        <meta name="docker" content="true">
-        <meta name="win32client" content="/BlocklyPropClient-setup-32.exe">
```

Comment:
The BlocklyProp Client download for Windows and MacOS will be available in the solo version of the app. Is something replacing the meta tags related to the Client downloads?

Reply:
Same, these need to be adjusted